artiq/coredevice/phaser.py

```diff
@@ -40,6 +40,16 @@
 PHASER_ADDR_DAC1_DATA = 0x28
 PHASER_ADDR_DAC1_TEST = 0x2c
 
+# servo registers
```

Comment:
Document where and when servo is supported. What's required. What happens if the gateware doesn't have it, what happens if the gateware does but it's not configured.